web-dashboards/README.md

```diff
@@ -79,6 +84,41 @@ To start both services:
 
 `docker-compose -f docker-compose.yml up kibiter`
 
+### Kibiter 6
+
+Since the first review of the project, Kibiter 6 is used to build the web dashboards.
+
+This version includes the 6.3.1 version for Elasticsearch and Kibana and the SearchGuard authentication plugin. 
+
+By default Kibiter is used in read only mode and, if you want to modify the web dashbboards, you need
+to login the user `admin` with password `admin`. 
```

Comment:
to login with user `admin` and password `admin`

Reply:
fixed